stpsf/trending.py

```diff
@@ -704,6 +715,13 @@ def single_measurement_trending_plot(
             print('   That measurement has a correction, therefore reading in post-move sensing data and SUR.')
             print(f'   Inferred prior correction ID is {prior_corr_id}')
         sur_fn = f'{prior_corr_id}_sur.xml'
+        if not os.path.exists(sur_fn):
+            # check for slight differences in SUR fn
+            import glob
+            try_sur_fns = glob.glob('?'+sur_fn[1:7]+"*.xml")
```

Comment:
It is better to add `*_sur.xml', otherwise the search will return a list with any xml file not only SUR, and the code will return an error when trying to use it. This happens in particular with the session data file which also have the same correction ID. Initially this happened to me because in the same folder I have the contents of the AP including the session data, the SUR and trail SUR, so the code was not picking the correct SUR.